packages/common/env/src/global.ts

```diff
@@ -32,6 +32,7 @@ export const runtimeFlagsSchema = z.object({
   enableInfoModal: z.boolean(),
   enableOrganize: z.boolean(),
   enableThemeEditor: z.boolean(),
+  enableEditorSettings: z.boolean(),
```

Comment:
add to https://github.com/toeverything/AFFiNE/blob/e01ce2cd9163dfe751c5e8979bfe5a632e0a9f3d/packages/common/infra/src/modules/feature-flag/constant.ts#L8